dvc/command/init.py

```diff
@@ -13,7 +13,7 @@ def run(self):
 
         try:
             self.repo = Repo.init(
-                ".",
+                self.args.cd,
```

Comment:
It occurred to me later maybe this would be better as a chdir in the CmdBaseNoRepo (w/e?) constructor?